src/Raven.Client/Documents/LinqExtensions.cs

```diff
@@ -1044,17 +678,16 @@ public static IOrderedQueryable<T> ThenByScoreDescending<T>(this IOrderedQueryab
         }
 
         /// <summary>
-        /// Returns the query results as a stream
+        ///     Returns the query results as a stream.
```

Comment:
Why did you add `    ` (tab) here and in other places?

Reply:
I think it makes the summary more readable for documentation changing purposes

Reply:
The problem is that this introduces inconsistency.

Reply:
Changed